### ISSUE TITLE
`cfssl sign` command does not expose the certificate's CA bundle

### ISSUE THREAD

**Issue by Feld0** [cloudflare/cfssl#779]
When using the `cfssl sign` CLI command to get a certificate signed by a remote cfssl server, it's useful to be able to get the CA bundle as well. The `cfssl sign` command, however, does not expose the API's "bundle = true" option; and calling the API directly does not work right now either due to #769.

Is there a different intended workflow for retrieving the bundle associated with a newly signed certificate? @lziest's comment on #769 suggests that this is an uncommon flow but I couldn't find another way to get a new certificate and its bundle in the same request.

**Comment by johanbrandhorst** [comment]
Bump, this is a pretty important issue I think. This is something supported by Hashicorp Vault out of the box and CFSSL should not be any worse.

**Comment by akamac** [comment]
Indeed, while api [does expose](https://github.com/cloudflare/cfssl/blob/master/doc/api/endpoint_sign.txt) `bundle` option, it's not available neither via `cfssl gencert`, nor `cfssl sign`.
Could this be fixed, please?

**Comment by hugoboos** [comment]
Adding a `Bundle` field to the `SignRequest` struct is fairly easy: https://github.com/cloudflare/cfssl/blob/8090bceefe6c9cc3b044197a8341bea2619d00a6/signer%2Fsigner.go#L49. Tested it myself. The API adds the bundle to the response.

The `Signer` interface has `Sign(req SignRequest) (cert []byte, err error)` defined: https://github.com/cloudflare/cfssl/blob/8090bceefe6c9cc3b044197a8341bea2619d00a6/signer%2Fsigner.go#L120
It returns only a cert. We could add the intermediates to that cert to not break the interface.
